src/ceph-disk/ceph_disk/main.py

```diff
@@ -1922,6 +1922,10 @@ def parser():
             help='unique OSD uuid to assign this disk to',
         )
         parser.add_argument(
+            '--crush-class',
```

Comment:
In the context CrushWrapper we use **device** but in the mon we use **osd**. In the context of **ceph osd crush** repeating **device** or even **crush** is redundant and we can just have **set-class** instead. I used **set-device-class** initially because the CrushWrapper function has the same name but in retrospect I think it was not a good idea.